### ISSUE TITLE
Self-learning GCP, Datalab, Python

### ISSUE THREAD

**Issue by ccasimiro9444** [bkkhack/hacknights#201]

**Comment by ccasimiro9444** [comment]
Using NYC yellow taxi dataset

**Comment by danfowler** [comment]
:eyes:

**Comment by danfowler** [comment]
GCP == Google Cloud Platform ? Cool : What is GCP?

**Comment by ccasimiro9444** [comment]
Yes, Google Cloud Platform. There are some large public datasets on Google's Bigquery, that could be imported into Google Cloud Datalab (similar to Jupyter Notebooks) and then I want to use Python to visually render the data.
I am from a stats background, so Python and GCP are kinda new to me. But saw this in action and hence will try to replicate some of it. But I am already stuck importing the Bigquery data into Datalab :) Hope you guys can help me figuring it out.

**Comment by ccasimiro9444** [comment]
my question on stackoverflow
https://stackoverflow.com/questions/44172105/load-bigquery-data-to-datalab

**Comment by ccasimiro9444** [comment]
SELECT
  pickup_datetime,
  dropoff_datetime
FROM
  `bigquery-public-data.new_york.tlc_yellow_trips_20*`

**Comment by kev-ho** [comment]
Go here
https://cloud.google.com/bigquery/public-data/nyc-tlc-trips

and click on the "goto the new york city dataset" button

**Comment by danfowler** [comment]
"""
#standardSQL
SELECT
pickup_datetime,
dropoff_datetime
FROM
\`bigquery-public-data.new_york.tlc_yellow_trips_20*\`
""")

**Comment by danfowler** [comment]
@grandpotato @ccasimiro9444 here is the notebook: https://github.com/danfowler/bkkhack-stuff/blob/master/nyc-taxi-data.ipynb

**Comment by ccasimiro9444** [comment]
Got it to run on datalab, just used Dan's code.
![screen shot 2017-05-26 at 00 15 49](https://cloud.githubusercontent.com/assets/20962719/26461900/dbfcf144-41a8-11e7-9c98-33ff7bee493f.png)

**Comment by danfowler** [comment]
Excellent! 💯 

By the way, I finally got datalab working on my Google account 😉 

Maybe, if you haven't gotten around to it by then, we work on the visualization piece at the next bkkhack.

**Comment by kev-ho** [comment]
Nice!

Unfortunately I've had 0 success so far.  So I'm going to just have to leave it here for now. :(

Next time I'll just clear everything and start from scratch and see if that helps.

**Comment by ccasimiro9444** [comment]
Sounds good. Gotta get the whole dataset into a dataframe first, the direct import from Bigquery to dataframe takes too long.
Maybe loading the table and then transforming will be faster.
Let's try that and the visualization out at the next hack.
